### DISCUSSION TITLE
APPRECIATION

### DISCUSSION THREAD

**Discussion by beejaycrypto** [Cyfrin/foundry-full-course-cu#1906] [General]
Hi tutors and fellow programmers.
I just started out taking this course but I must confess that it is so value-packed. Even though I am a newbie, the lessons were very explanatory. I look forward to coming out soon as a Dev.
I just got to lesson 2 by the way, and I have started interacting with Remix.
Thank you.

**Comment by EngrPips** [comment]
Hello @beejaycrypto, I  wish you the best on your journey to become a web3 develper. Keep going and don't stop.
